src/types/book.ts

```diff
@@ -8,15 +8,20 @@ export class Book implements BookType {
     category: CategoryType;
     bestInCategory: boolean;
     amazonProductID: string;
-    amazonProductURL: string;
     amazonImageURL: string;
     hasInventory: boolean;
 
     constructor(book: BookType) {
         Object.assign(this, book);
 
-        this.amazonProductURL = `https://www.amazon.com/dp/${this.amazonProductID}?tag=${AMAZON_TRACKING_ID}&language=en_US`;
+        this.amazonImageURL = `/images/products/${this.amazonProductID}.jpg`;
```

Comment:
No, can you please put amazonImageURL back, but as a getter?

Order:
- product id
- amazon image url
- image url